test/spec/core/CanvasSpec.js

```diff
@@ -951,6 +951,126 @@ describe('Canvas', function() {
 
   });
 
+  describe('#scrollToElement', function() {
+
+    beforeEach(function() {
+      container = TestContainer.get(this);
+    });
+    beforeEach(createDiagram({ canvas: { width: 500, height: 500 } }));
+
+
+    it('scrolls element into view', inject(function(canvas) {
+
+      // given
+      var element = { x: 650, y: 650, width: 50, height: 50 };
```

Comment:
I don't like that we're testing with fake elements when the API expects an element? Or does it not? If it accepts bounds then let's make that clear.

Reply:
You are right, I now use real shapes instead of a fake element :+1: